config/locales/en.yml

```diff
@@ -852,3 +852,22 @@ en:
       statements:
         ecf_statement_selector:
           submit: "View"
+
+  support_form:
+    introduction:
+      unspecified: "If you have a question about the Early Career Framework (ECF) service, you can contact the DfE using this form."
+      change-participant-lead-provider: "Complete this form to request a change to %{participant_name}’s registered training provider."
+    instructions:
+      unspecified_html: "Enter your support query and any relevant information below"
+      change-participant-lead-provider_html: |
+        Tell us:
```

Comment:
```suggestion
        <p class="govuk-body">Tell us:</p>
```

Reply:
Moving this inside here means duplicating it every time rather than just having it one in the template. Plus, that's templating rather than copy which doesn't really belong in the locale file.